src/org/zaproxy/zap/extension/scripts/ConsolePanel.java

```diff
@@ -185,6 +190,25 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		return stopButton;
 	}
 
+    private ZapToggleButton getAutoCompleteButton() {
+        if (autoCompleteButton == null) {
+            autoCompleteButton = new ZapToggleButton();
+            autoCompleteButton.setIcon(DisplayUtils.getScaledIcon(AUTO_COMPLETE_ICON));
+            autoCompleteButton.setSelectedIcon(DisplayUtils.getScaledIcon(AUTO_COMPLETE_ICON));
+            autoCompleteButton.setToolTipText(Constant.messages.getString("scripts.toolbar.tooltip.autocomplete.disabled"));
+            autoCompleteButton.setSelectedToolTipText(Constant.messages.getString("scripts.toolbar.tooltip.autocomplete.enabled"));

```

Comment:
Should the tool tips be the other way around? (It should indicate what will happen if clicked, not the current state?)

Reply:
Was going to have them like that, but saw the script and scroll lock buttons show the current state so copied them for consistency...